src/app/features/container/containers/headers/dialog-header.tsx

```diff
@@ -7,9 +7,10 @@ import { CloseIcon, IconButton } from '@leather.io/ui';
 interface DialogHeaderProps {
   onClose?(): void;
   title?: ReactNode;
+  variant?: 'default' | 'large';
 }
 
-export function DialogHeader({ onClose, title }: DialogHeaderProps) {
+export function DialogHeader({ onClose, title, variant = 'default' }: DialogHeaderProps) {
```

Comment:
I changed this to accept a variant as previously we were using another variant in `Header` - `bigTitle`. It's used in the `Send` flow.  

![Screenshot 2024-07-09 at 15 16 08](https://github.com/leather-io/extension/assets/2938440/91317420-6295-4b3f-9672-870937e4b1f8)

I'll move this component to the mono-repo to be located with `Dialog`